draft-ietf-quic-transport.md

```diff
@@ -2859,13 +2859,16 @@ valid frames? -->
 
 ### Sending ACK Frames
 
-An endpoint MUST NOT send more than one packet containing only an ACK frame per
-received packet that contains frames other than ACK and PADDING frames.
-An endpoint MUST NOT send a packet containing only an ACK frame in response
-to a packet containing only ACK or PADDING frames, even if there are packet
-gaps which precede the received packet. This prevents an indefinite feedback
-loop of ACKs. The endpoint MUST however acknowledge packets containing only
-ACK or PADDING frames when sending ACK frames in response to other packets.
+Packets containing only ACK frames are not congestion controlled, so there are
+limits on how frequently they can be sent.  An endpoint MUST NOT send more than
+one packet containing only an ACK frame per received ACK-eliciting packet
+(one containing frames other than ACK and/or PADDING).  An endpoint MUST NOT
+send a packet containing only an ACK frame in response to a non-ACK-eliciting
+packet (one containing only ACK and/or PADDING frames), even if there are
+packet gaps which precede the received packet. This prevents an indefinite
+feedback loop of acknowledgements, which may prevent the connection from ever
+becoming idle. The endpoint MUST however acknowledge non-ACK-eliciting packets
+when sending ACK frames in response to other packets.
```

Comment:
Is "The endpoint MUST however acknowledge non-ACK-eliciting packets when sending ACK frames in response to other packets." necessary?  This is simply saying that ACK frames should ACK received packets.

Reply:
I think so - it may not be technically required, but it would be unclear to a new reader whether non-ack-eliciting packets should be acked, and if so, how.

Reply:
Oh, I think I suggested that before and @ianswett said the peer might want to control when to receive ACK's.

Reply:
We can say it differently, but it's important to say somewhere that non-ACK-eliciting packets are ACKed when an ACK is sent.

Reply:
that Oh, ... comment landet the wrong place - I responded elsewhere

> What if change the requirement to simply not ACK'ing ACK's too often. That would also do away with PING feedback loops. It seems that we are just moving the problem.

and this is what the comment was an update to.